apiary.apib

```diff
@@ -1002,6 +1002,10 @@ Following operations will be **supported based on the field**. You can also comb
         Supported operations - `eq`, `in`, `lt`, `lte`, `gt`, `gte`
         + created_at (string, optional) - Date on which the credentials have been created. Format: YYYY-MM-DD
         Supported operations - `eq`, `in`, `lt`, `lte`, `gt`, `gte`
+    + sort (object, optional)
+        + String key/value pair for sorting based on a field and order.
+        Supported fields - `id`, `recipient_name`, `group.name`, `issued_on` and `expired_on`.
+        Supported orders - `asc`, `desc`
```

Comment:
Subjective - I might prefer calling them `Supported keys` & `Supported values` since we mention it as "String key/value pair".

Reply:
That's a good improvement. Just updated it, thanks.